irohad/synchronizer/impl/synchronizer_impl.cpp

```diff
@@ -66,41 +89,56 @@ namespace iroha {
               and validator_->validateChain(chain, *storage)) {
             mutable_factory_->commit(std::move(storage));
 
-            return {chain, SynchronizationOutcomeType::kCommit};
+            return {chain, SynchronizationOutcomeType::kCommit, {}};
```

Comment:
Looks like round from `GateObject` is lost, so it should be passed here as well.